src/GeocodeSharp/Google/GeocodeClient.cs

```diff
@@ -28,7 +28,7 @@ public GeocodeClient()
         public GeocodeClient(string apiKey)
         {
             _apiKey = apiKey;
-            _baseUrl = string.Format("https://maps.googleapis.com/maps/api/geocode/json?key={0}", Uri.EscapeDataString(_apiKey));
+            _baseUrl = string.Format("https://maps.googleapis.com/maps/api/geocode/json?key={0}&", Uri.EscapeDataString(_apiKey));
```

Comment:
Added ampertsand here, otherwise the address parameter gets added directly onto the end of the API key.